packages/ui-components/src/components/Pagination/Pagination.stories.js

```diff
@@ -28,22 +28,26 @@ export default {
   ],
 }
 
-const Template = ({ currentPage, pages, ...args }) => {
+const Template = (args) => {
+  return <Pagination {...args} />
+}
+
+const UncontrolledTemplate = ({ currentPage, pages, ...args }) => {
   const [page, setPage] = React.useState(currentPage)
   const prev = React.useCallback(() => setPage(page > 1 ? page - 1 : 1), [page])
   const next = React.useCallback(() => setPage(!pages || page < pages ? page + 1 : pages), [page])
 
-  return <Pagination {...args} pages={pages} currentPage={page} onPressPrevious={prev} onPressNext={next} />
+  return <Pagination {...args} totalPages={pages} currentPage={page} onPressPrevious={prev} onPressNext={next} />
 }
 
-Template.propTypes = {
+UncontrolledTemplate.propTypes = {
   currentPage: PropTypes.number,
   pages: PropTypes.number,
 }
 
 export const Default = {
   render: Template,
-  args: {},
+  args: { variant: "default" },
```

Comment:
It only seems to be disabled because the values “currentPage” and “pages” are not set. As soon as you can neither click forward nor back, the buttons are in the disabled state. To correct this, I have added the above value.